stories/internal/MeasureNumberFormat.tsx

```diff
@@ -0,0 +1,104 @@
+// (C) 2020 GoodData Corporation
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import { ISeparators } from "@gooddata/numberjs";
+import { screenshotWrap } from "@gooddata/test-storybook";
+
+import { IFormatPreset, IToggleButtonProps } from "../../src/components/menu/measureNumberFormat/typings";
+import { screenshotTarget } from "../utils/wrap";
+import { MeasureNumberFormat } from "../../src/components/menu/measureNumberFormat/MeasureNumberFormat";
+import { IFormatTemplate } from "../../src/components/menu/measureNumberFormat/CustomFormatTemplates";
+
+storiesOf("Internal/Measure Number Format", module).add("Measure Number Format", () => {
```

Comment:
The component is in the "menu" dir in the source code so I guess it should be in the "menu" dir in storybook too.

Reply:
Possibly we could find better directory for it as I'm not sure "menu" is that great. Originally I had menu in name of the components but removed it. I wanted to move it in different directory after mutual discussion.

Reply:
I was unable to come up with a proper location so I've created it's own top-level directory instead.